<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="sudhakar-PC" failures="1" tests="3" name="My Test" time="0.076" errors="0" timestamp="09 Mar 2019 10:07:13 GMT">
  <testcase classname="SamSung.Actions124" name="TC02" time="0.026">
    <failure type="java.lang.AssertionError" message="Write your custom error message">
      <![CDATA[java.lang.AssertionError: Write your custom error message
at SamSung.Actions124.TC02(Actions124.java:78)
at SamSung.TestNGMainClass.main(TestNGMainClass.java:89)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- TC02 -->
  <testcase classname="SamSung.Actions124" name="TC07" time="0.0"/>
  <testcase classname="SamSung.Actions124" name="TC08" time="0.0"/>
</testsuite> <!-- My Test -->
